test/features/basic-features/mandos/big_floats.scen.json

```diff
@@ -1,5 +1,6 @@
 {
     "gasSchedule": "v3",
+    "traceGas": true,
```

Comment:
This is for debugging only, should be turned off when publishing test.

Reply:
done